spec/graphql/directive_spec.rb

```diff
@@ -8,6 +8,7 @@
         # plain fields:
         skipFlavor: flavor @skip(if: true)
         dontSkipFlavor: flavor @skip(if: false)
+        dontSkipDontIncludeFlavor: flavor @skip(if: false), @include(if: false)
```

Comment:
I see your point on testing presence, but could you also add a test for absence? To be sure that skip still "does its thing" even if `@include` is present